torch/_dynamo/variables/tensor.py

```diff
@@ -1128,6 +1128,20 @@ def create(cls, tx, proxy, sym_num=None, **options):
             sym_num = int(sym_num) if isinstance(sym_num, sympy.Integer) else sym_num
             return ConstantVariable.create(sym_num)
 
+        def _is_unbacked_symbol(s):
+            from torch.fx.experimental.symbolic_shapes import (
+                free_unbacked_symbols,
+                is_symbolic,
+            )
+
+            return (
+                is_symbolic(s)
+                and isinstance(s.node.expr, sympy.Symbol)
+                and len(free_unbacked_symbols(s.node.expr)) > 0
+            )
+
+        if _is_unbacked_symbol(sym_num):
+            tx.output.track_unbacked_symbols(sym_num, proxy)
```

Comment:
Why do we need this extra code?

Reply:
We need to be able to find the proxy for unbacked symbols in parent (to return lifted_freevars for subgraph to connect with parent graph) when we lift symbols created outside of the subgraph as subgraph inputs. 

Unbacked symbols can be created as a return value of some function calls so we cannot do this in wrap_tensor.

Reply:
Do you have a small test case that exercises this code? I want to step through an example (I'm not understanding how we get here)